projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/history/event-log/event-log-table.component.ts

```diff
@@ -23,8 +23,16 @@ export class EventLogTableComponent implements OnInit {
 
   public isPartOfCaseTimeline = false;
 
+  public isUserExternal: boolean;
+
+  constructor(
+    private sessionStorage: SessionStorageService
+  ){}
+
   public ngOnInit(): void {
     this.isPartOfCaseTimeline = this.onCaseHistory.observers.length > 0;
+    this.isUserExternal = JSON.parse(this.sessionStorage.getItem('userDetails')).roles.includes('pui-case-manager');
+    console.log(this.isUserExternal);
```

Comment:
@Josh-HMCTS do we need this console log?

Reply:
no - remove please